content/post/articles/017_Power_BI/index.md

```diff
@@ -2,7 +2,7 @@
 title: "Power BI"
 date: 2021-03-15
 categories: ["مقالات"]
-tags: ["PowerBI", "DataVisualization", "DataScience", "Tools"]
+tags: ["PowerBI", "Data Visualization", "Data Science", "Tools"]
```

Comment:
I already updated those so no need to do them again.

Reply:
Ok

Reply:
Then please undo these changes or open a new PR :)